airgun/entities/computeprofile.py

```diff
@@ -31,6 +35,18 @@ def delete(self, entity_name):
         view.table.row(name=entity_name)['Actions'].widget.fill('Delete')
         self.browser.handle_alert()
 
+    def list_computeprofiles(self, entity_name, computeprofile_name):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.compute_profiles.table.row(compute_profile=computeprofile_name)[
+            'Compute profile'].widget.click()
+        view.submit.click()
+
+    def read_computeprofile(self, entity_name, computeprofile_name):
```

Comment:
same here. btw i'd also implement `read()` helper to be able to fetch compute profile details